### DISCUSSION TITLE
What is `temporary_folder` used for?

### DISCUSSION THREAD

**Discussion by tbsbd** [danielhrisca/asammdf#898] [Q&A]
What is `temporary_folder` used for?

The description is not helpful.
> temporary_folder: str
>   default folder for temporary files
>   New in version 7.0.0.

Does setting this enables that temporary files are written and performance is improved?
Or does this just change the location of temporary files which are anyways written?

**Comment by danielhrisca** [comment]
if there is limited space on the partition that is normally used by the OS for temporary files, then you can point it to some other folder on a partition that has more free space
